usaspending_api/database_scripts/job_archive/recompute_all_awards.py

```diff
@@ -44,7 +43,7 @@
     CONNECTION_STRING = environ["DATABASE_URL"]
 except Exception:
     print("SET env var DATABASE_URL!!!\nTerminating script")
-    raise SystemExit(1)
+    raise SystemExit(100)
```

Comment:
Why status 100? Is this meaningful to Jenkins?

Reply:
More like "why not?"
No significance.

Reply:
Giving non-standard feedback to something well established like program exit codes can risk confusion. "1" is a standard "The program broke, and this code has no special information why" response, while "100" seems like something specific and intentional. No harm besides making some poor guy do some pointless googling though.

Reply:
It is useful to standardize on scripts to use the same exit codes for similar errors. That way someone knowledgeable with the script can use the exit code to reference where and why it failed.

We don't do that at the moment. 100 isn't known to be tied to a specific meaning, so it won't be confused with existing patterns. It would be trivial to set them all to 1 if you believe it is too confusing